test/cloudburst.yml

```diff
@@ -0,0 +1,127 @@
+# Advanced configuration file for Cloudburst
```

Comment:
Is this file supposed to be here?

Reply:
yes, they are used for automated testing so we can be sure the Jackson config is correct

Reply:
not at all. sorry for that